### PR TITLE
Feature/2017 10 24 blogpost kickstarters 2017

### PR DESCRIPTION
**Pull request by JeroenVanWolput**
Added the blogpost and jeroen van wolput as an author

### CODE REVIEW

**Comment by JeroenRombouts** [comment]
Are you waiting for a review? If so: you can add reviewers on the right....

File: _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+layout: post
+authors: ["jeroen_van_wolput"]
+title: 'NCORE KICKSTARTERS 2017'
+image: img/kickstarters2017/banner.jpg
+tags: [kickstarters]
+category: Kickstarters
+comments: true
+---
+
+**The first day of a journey will always be remembered. On august 1st over 40 open minded and enthusiastic kickstarters embarked on a journey into the unknown. A journey filled with mountains to climb and rivers to cross. One of those teams sprouted a team of 6 dedicated travelers with as destination the N-Core unit. As we travelled along we filled our bags with knowledge and experience guided by our coaches and managers. After almost 2 months we reached our destination, changed our clothes and re-embarked on a new development journey.**
```
Comment (jensg) [inline _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md]: > **One of those teams** sprouted a team of 6 dedicated travelers
There is a reference to a team without mentioning a team first.

File: _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* TFVC and Git
+* Xamarin (Forms)
+* Backlog refinement + sprint planning dev case
+* Communication Training
+* Brand yourself
+
+### 4. Group
+As mentioned before, having fun was just as important as work! We are facing each other 8 hours a day. So team spirit is a big thing. Since the first day we met there was a connection. We laughed and told some interesting stories about ourselves on our first day at Ordina. 
+Through the studies, we helped each other out and nobody was left behind. Nobody didn’t mind to give some extra explanation after the hours or to repeat the same thing a million times. To give a conclusion about the group we can give you a few characteristics about our group. We love to have fun, love to life and love to program.
+![The new Kickstarters](/img/kickstarters2017/kickstartersGroup.JPG "The new Kickstarters")
```
Comment (jensg) [inline _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md]: Maybe leave out this image?

**Review by jensg — CHANGES_REQUESTED** [review]

File: _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md (hunk 2 above)
Comment (JeroenVanWolput) [reply _posts/2017-10-24-jeroen-van-wolput-kickstarters-2017.md]: Leave the image with Sami in?

**Review by jensg — APPROVED** [review]

**Pull request merged**